mypy/stats.py

```diff
@@ -332,7 +332,8 @@ def type(self, t: Optional[Type]) -> None:
             return
 
         if isinstance(t, AnyType) and is_special_form_any(t):
-            # This is not a real Any type, so don't collect stats for it.
+            # If there is an error in special form definition. it will override TYPE_PRECISE
```

Comment:
```suggestion
            # If there is an error in special form definition, it will override TYPE_PRECISE
```

Do we actually have a test for this?

Reply:
no we don't and actually after checking, i found out the comment is wrong.

Reply:
Is there an update that should be made?

Reply:
i checked this for a while, and i have a couple of questions:  
1. what is the desired behavior? (right now typeAlias lines are marked as unanalysed and thus its rendered Red)
2. this PR will fix the original issue but i'm afraid it may add other undesirable side effects that we don't have tests for them yet.    

if some one could help me to create test cases for both correct and incorrect typeAlias exprs. i may be able to fix this PR.  
thanks.

Reply:
This PR is fine to land, if this comment is updated. Can you replace the comment with a TODO comment like this: 

`# TODO: What if there is an error in special form definition?`

I don't think that errors is special forms are super important to test, since it's reasonable to expect that there is typically a clean mypy run before generating a report.